.github/actions/bump-golang/action.yml

```diff
@@ -36,13 +36,12 @@ runs:
         uses: updatecli/updatecli-action@3a8785d88ec4fa03d86521a181f37c0e74627463 # v2.64.0
 
       - name: Run Updatecli in Apply mode
-        run: updatecli ${{ env.COMMAND }} --config ./.github/updatecli.d/bump-golang.yml
+        run: updatecli ${{ env.COMMAND }} --config ./.github/updatecli.d/bump-golang.yml --debug
```

Comment:
Do we need debug permanently?

Reply:
I thought the same; IMO, it might help in the future, but I have no strong opinion.

Reply:
I'd suggest going the safe path, let's remove it since the output is publicly accessible and we don't know what it could potentially show (probably nothing, but as I said, let's keep safe).

Reply:
GitHub secrets cannot be exposed at all, they are always masked as far as I know.
Sure

Reply:
```suggestion
        run: updatecli ${{ env.COMMAND }} --config ./.github/updatecli.d/bump-golang.yml
```